File: P1-Linguagem_de_Programação_I/Desafios dos Roteiros - Python/desafio-roteiro06/versao-modular/ex07.py
def classificar_nota(nota):

    if nota >= 90:
        return "A"
    elif nota >= 80:
        return "B"
    elif nota >= 70:
        return "C"
    elif nota >= 60:
        return "D"
    else:
        return "E"

nota = int(input("Digite a nota: "))

resultado = classificar_nota(nota)

print("Categoria:", resultado)

File: P1-Linguagem_de_Programação_I/Desafios dos Roteiros - C++/desafio-roteiro06.cpp/versao-modular/ex07.cpp
#include <iostream>
using namespace std;

char classificar_nota(int nota) {

    if (nota >= 90) return 'A';
    else if (nota >= 80) return 'B';
    else if (nota >= 70) return 'C';
    else if (nota >= 60) return 'D';
    else return 'E';
}

int main() {

    int nota;

    cout << "Digite a nota: ";
    cin >> nota;

    cout << "Categoria: " << classificar_nota(nota);

    return 0;
}